.travis.yml

```diff
@@ -1,33 +1,41 @@
 language: android
-
+sudo: required
 jdk: oraclejdk8
 
 env:
   global:
-    - MALLOC_ARENA_MAX=2
-    - ADB_INSTALL_TIMEOUT=10
-  matrix:
-    - ANDROID_TARGET=android-22  ANDROID_ABI=armeabi-v7a
+    - ANDROID_API_LEVEL=28
+    - ANDROID_BUILD_TOOLS_VERSION=28.0.3
+    - ANDROID_ABI=armeabi-v7a
 
 android:
   components:
     - tools
-    - build-tools-25.0.3
-    - android-25
-    - android-24
-    - android-22
-    - android-23
-    - sys-img-armeabi-v7a-android-22
-    - extra-google-m2repository
+    - platform-tools
+    - tools
```

Comment:
You have `tools` twice.

Reply:
👍 good catch, initially last time I touched this file it was the only way to make it work! but apparently that bug was fixed :D